src/menu/pause/restart_button.c

```diff
@@ -0,0 +1,49 @@
+/*
+** EPITECH PROJECT, 2022
+** rpg
+** File description:
+** restart button 
+*/
+
+#include "my_rpg.h"
+#include "main_menu.h"
+
+static void restart_player_state(window_t *win)
+{
+    player_t *player = dico_t_get_value(win->components, PLAYER);
+    any_t *stats = NULL;
+    any_t *life = NULL;
+    any_t *energy = NULL;
+
+    if (player == NULL) {
```

Comment:
Check if player->obj == null
Because it can in scene transition when change object or smth line this